spec/jobs/push_pom_to_delius_job_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe PushPomToDeliusJob, :allocation, type: :job, versioning: true do
+  let(:offender) { build(:nomis_offender) }
+  let(:offender_no) { offender.fetch(:offenderNo) }
+  let(:pom) { build(:pom) }
+
+  before do
+    stub_auth_token
+    stub_community_set_pom(offender)
```

Comment:
This line (stub_community_set_pom) looks wrong at this level as it is intercepted in tests